cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/KafkaRoller.java

```diff
@@ -701,27 +702,38 @@ private void checkIfRestartOrReconfigureRequired(NodeRef nodeRef, boolean isCont
     /* test */ void dynamicUpdateBrokerConfig(NodeRef nodeRef, Admin ac, KafkaBrokerConfigurationDiff configurationDiff, KafkaBrokerLoggingConfigurationDiff logDiff)
             throws ForceableProblem, InterruptedException {
         boolean isLog4j2 = KafkaVersion.compareDottedVersions(kafkaVersion.version(), "4.0.0") >= 0;
-        Map<ConfigResource, Collection<AlterConfigOp>> updatedConfig = new HashMap<>(2);
+        Map<ConfigResource, Collection<AlterConfigOp>> brokerUpdatedConfig = new HashMap<>(2);
```

Comment:
If we are really renaming it ... would `updatedPerBrokerConfig` and `updatedClusterWideConfig` be better names?